task/06-conditions-n-loops-tasks.js

```diff
@@ -270,7 +338,23 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+
+    let str = num.toString();
+    let sum = 0;
+    let final = 0;
+    
+    for (let i = 0; i < str.length; i++) {        
+        sum += Number(str[i]);        
+    }
+    if (sum > 9) {
+        let sumStr = sum.toString();
+        for (let j = 0; j < sumStr.length; j++) {
+
+            final += Number(sumStr[j]);            
+        }
+        return final;
+    }
+    else return sum;
```

Comment:
Looks good, but you can do the same much easier with array methods (in 3 rows)  :) Please try to improve your answer